GNUmakefile

```diff
@@ -24,6 +24,7 @@ install: ${PROG} ${MAN}
 	chmod ${BINMODE} ${DESTDIR}${BINDIR}/${PROG}
 	cp -f doas.1 ${DESTDIR}${MANDIR}/man1
 	cp -f doas.conf.5 ${DESTDIR}${MANDIR}/man5
+	cp -f doas.d.5 ${DESTDIR}${MANDIR}/man5
```

Comment:
nit: could the installation of this file be optional?

Reply:
Not sure how to integrate it with the build system.  Open to suggestions.

Reply:
Do suggestions from the peanut gallery count? If `WITHOUT_CONFDIR` is written out to config.mk as either `true` or `false`, then the rule can say `${WITHOUT_CONFDIR} || cp ...`. Alternatively, if it is written out as `yes` or empty string, it can instead say `[ -n "${WITHOUT_CONFDIR}" ] || cp ... `; that feels trickier, but is more in line with how `configure` works right now.

For reference, when Automake does this sort of thing, it substitutes either the empty string or the comment character in the beginning of the line, which yields cleaner output from `make` but is also something of a hack and doesn’t allow for a neatly separated `config.mk`.